include/inviwo/core/util/stringconversion.h

```diff
@@ -269,6 +270,13 @@ constexpr std::string_view trim(std::string_view str) noexcept {
  */
 IVW_CORE_API bool iCaseEndsWith(std::string_view str, std::string_view suffix);
 
+constexpr auto fmtHelp = IVW_UNINDENT(R"(
+    To locate the error try breaking on fmt::format_error exceptions,
+    or put a breakpoint in fmt::detail::error_handler::on_error.
```

Comment:
Yes, but it is hard to figure out what is wrong when you get a message "invalid format" and it can be any format string in all of Inviwo that is to blame... Might at the relevant MVSC and GCC commands to at some point...